examples/model_compress/pruning/speedup/speedup_yolov3.py

```diff
@@ -0,0 +1,36 @@
+import torch
+from pytorchyolo import models
+
+from nni.compression.pytorch import ModelSpeedup
+from nni.algorithms.compression.pytorch.pruning import L1FilterPruner, LevelPruner
+from nni.compression.pytorch.utils import not_safe_to_prune
```

Comment:
seems this function has not updated.

Reply:
Yes, this pr depends on https://github.com/microsoft/nni/pull/3462

Reply:
got it